site/data/kapp-controller/docs/kapp-controller-develop-toc.yml

```diff
@@ -27,6 +27,8 @@ toc:
         url: /packaging-tutorial
       - page: "Consuming Packages using the CLI"
         url: /kctrl-package-tutorial
+      - page: Packaging upstream artifacts with CLI
+        url: /kctrl-upstream-authoring-tutorial
```

Comment:
may be? kctrl-upstream-authoring-tutorial => kctrl-package-authoring-of-upstream

Reply:
Done!